Modify the module Calendar to prevent notice on Day display and Week display ($view->style_options['groupby_times'] missing)
Index: /modules/field_ui/field_ui.js
=======================================================================================================================================
--- /modules/field_ui/field_ui.js	2012-03-14 14:28:24.000000000 +0100
+++ /modules/field_ui/field_ui.js	2012-03-14 14:38:35.000000000 +0100
@@ -90,7 +90,8 @@
       html += '<option value="' + value + '"' + (is_selected ? ' selected="selected"' : '') + '>' + text + '</option>';
     });
 
-    $(this).html(html).attr('disabled', disabled ? 'disabled' : '');
+    //$(this).html(html).attr('disabled', disabled ? 'disabled' : '');
+    $(this).html(html).attr('disabled', disabled ? 'disabled' : false);
   });
 };
 
